ibis/expr/operations.py

```diff
@@ -349,6 +349,17 @@ def output_type(self):
             return rules.shape_like(arg, 'double')
 
 
+class Clip(ValueOp):
+    input_type = [number(name='lower', allow_boolean=False, optional=True),
+                  number(name='upper', allow_boolean=False, optional=True)]
```

Comment:
just returns the original. not great. I could raise if not at least 1 is passed.